### DISCUSSION TITLE
vaapi support ?

### DISCUSSION THREAD

**Discussion by error105** [weltenwort/frigate-synology-dsm7#72] [Q&A]
In the latest version of image, there is a vaapi support ? Because if im use hw-acel or not, always i have about 35% CPU load :(
I use:
ffmpeg:
  hwaccel_args:
    - -hwaccel
    - vaapi
    - -hwaccel_device
    - /dev/dri/renderD128
    - -hwaccel_output_format
    - yuv420p
On my Intel® Core™ i7-7567U

**Comment by baylanger** [comment]
The variable name was changed by frigate team.  Did you update you configuration?

**Comment by baylanger** [comment]
https://github.com/blakeblackshear/frigate/releases/tag/v0.11.0-rc3

Read Changes rc1

**Comment by error105** [comment]
I have that changes, i use also correct config file, but still its looks like my iGPU do nothing :(

vainfo:
root@Frigate:/opt/frigate# vainfo                                                                                                
error: XDG_RUNTIME_DIR not set in the environment.                                                                               
error: can't connect to X server!                                                                                                
libva info: VA-API version 1.10.0                                                                                                
libva info: User environment variable requested driver 'i965'                                                                    
libva info: Trying to open /usr/lib/x86_64-linux-gnu/dri/i965_drv_video.so                                                       
libva info: Found init function __vaDriverInit_1_8                                                                               
libva info: va_openDriver() returns 0                                                                                            
vainfo: VA-API version: 1.10 (libva 2.10.0)                                                                                      
vainfo: Driver version: Intel i965 driver for Intel(R) Kaby Lake - 2.4.1                                                         
vainfo: Supported profile and entrypoints                                                                                        
      VAProfileMPEG2Simple            : VAEntrypointVLD                                                                          
      VAProfileMPEG2Simple            : VAEntrypointEncSlice                                                                     
      VAProfileMPEG2Main              : VAEntrypointVLD                                                                          
      VAProfileMPEG2Main              : VAEntrypointEncSlice                                                                     
      VAProfileH264ConstrainedBaseline: VAEntrypointVLD                                                                          
      VAProfileH264ConstrainedBaseline: VAEntrypointEncSlice                                                                     
      VAProfileH264ConstrainedBaseline: VAEntrypointEncSliceLP                                                                   
      VAProfileH264Main               : VAEntrypointVLD                                                                          
      VAProfileH264Main               : VAEntrypointEncSlice                                                                     
      VAProfileH264Main               : VAEntrypointEncSliceLP                                                                   
      VAProfileH264High               : VAEntrypointVLD                                                                          
      VAProfileH264High               : VAEntrypointEncSlice                                                                     
      VAProfileH264High               : VAEntrypointEncSliceLP                                                                   
      VAProfileH264MultiviewHigh      : VAEntrypointVLD                                                                          
      VAProfileH264MultiviewHigh      : VAEntrypointEncSlice                                                                     
      VAProfileH264StereoHigh         : VAEntrypointVLD                                                                          
      VAProfileH264StereoHigh         : VAEntrypointEncSlice                                                                     
      VAProfileVC1Simple              : VAEntrypointVLD                                                                          
      VAProfileVC1Main                : VAEntrypointVLD                                                                          
      VAProfileVC1Advanced            : VAEntrypointVLD                                                                          
      VAProfileNone                   : VAEntrypointVideoProc                                                                    
      VAProfileJPEGBaseline           : VAEntrypointVLD                                                                          
      VAProfileJPEGBaseline           : VAEntrypointEncPicture                                                                   
      VAProfileVP8Version0_3          : VAEntrypointVLD                                                                          
      VAProfileHEVCMain               : VAEntrypointVLD                                                                          
      VAProfileHEVCMain10             : VAEntrypointVLD                                                                          
      VAProfileVP9Profile0            : VAEntrypointVLD                                                                          
      VAProfileVP9Profile2            : VAEntrypointVLD
